.common-ci.yml

```diff
@@ -112,10 +112,6 @@ trigger-pipeline:
   variables:
     DIST: rhel9
 
-.dist-centos7:
```

Comment:
Do we want to add a note to the `centos7` folder in the repo to indicate that it is no longer supported?